### ISSUE TITLE
Can't open a terminal frame using emacsclient

### ISSUE THREAD

**Issue by Stebalien** [ch11ng/exwm#241]
I'd like to be able to use Emacs as my "window manager" in both TTYs *and* Xorg. However, when I run `emacsclient -t`, the terminal frame appears only briefly before it disappears. In my emacs `*Messages*` buffer I see:

```
(run-hook-with-args exwm-manage--remove-frame #<frame F3 0xaa4fd68>) signaled (wrong-type-argument stringp nil)
```

Note: I'm using emacs 26 (recent git) if it makes a difference.

**Comment by ch11ng** [comment]
Fixed in 7a5bb1156d07848e465f55fd31f8ec755b52aa01.

**Comment by Stebalien** [comment]
Thanks!
